packages/slice-machine/server/tsconfig.json

```diff
@@ -23,5 +24,6 @@
       "@models/*": ["../lib/models/*"]
     }
   },
-  "include": ["./**/*.ts", "./**/*.js", "../lib/**/*.ts", "../lib/**/*.js"]
+  "exclude": ["../build"],
```

Comment:
does adding build change anything?

Reply:
If a file from the build is imported directly then it avoid to try to transpile it. (we had this for the last start script).